fpga_interchange/fasm_generators/generic.py

```diff
@@ -334,15 +341,17 @@ def fill_pip_features(self, pip_feature_format, extra_pip_features,
                     wire1_id = self.device_resources.string_index[wire1]
 
                     pip = tile_type.pip(wire0_id, wire1_id)
+                    tile = tile_info.sub_tile_prefices[pip.subTile]
```

Comment:
I think that `tile` should be changed only in case there is a subTile corresponding to a `pip`, otherwise it would conflict with the `segment.tile`.

Reply:
I'm not sure I quite understand? `sub_tile_prefices` automatically contains the tile name as the first entry if subtiles are not being used: https://github.com/SymbiFlow/python-fpga-interchange/pull/89/files#diff-e5aa6b3db87bb1d0562eaf2c250aff020411809ee38368c5f7c84940c78d7d85R707 so this should work in all cases

Reply:
Right, I missed that, if no sub-tiles are present, the first entry is the tile itself.